tests/xgboost/test_xgboost_model_export.py

```diff
@@ -594,3 +595,18 @@ def test_model_log_with_signature_inference(xgb_model, xgb_model_signature):
 
     mlflow_model = Model.load(model_uri)
     assert mlflow_model.signature == xgb_model_signature
+
+
+def test_model_without_signature_predict(xgb_model):
```

Comment:
Does this test come from universe?

Reply:
Nope, the R test fails so I try to repro it in python. Universe doesn't include this test so their update of using np.array to convert data when parsing works fine.